outline.md

```diff
@@ -0,0 +1,17 @@
+## Motivation
+
+Introduce the document by describing the motivations for producing a consolidated set of software citation principles. This section is basically an overview of the different use cases/scenarios where it would be desirable to have a way to cite software.
+
```

Comment:
What's the best way of presenting the overview of the use/cases such that it doesn't take too long to get to the section on Software citation principles, and also so that it is easy to checkoff when you're justifying the principles? Can this be presented in a tabular form?